user_manual/files/file_drop.rst

```diff
@@ -1,60 +1,42 @@
-==========================================
-Making Anonymous Uploads (Enterprise Only)
-==========================================
-
-If your ownCloud administrator (Enterprise edition only) has enabled the 
-Files Drop application, you may create your own special upload directory so that 
-other people can upload files to you without having to log in to the server, and 
-without being an ownCloud user. They will not be allowed to see the contents of 
-this directory, or to make any changes. This is an excellent alternative to 
-sending large attachments via email, using an FTP server, or using commercial 
-file-sharing services.
+===========================
+Making Anonymous Uploads
+===========================
+
+If your Nextcloud administrator has enabled the Files Drop application, you may
```

Comment:
Yes I know. And I have changed the text completly with new screenshots...
... but forgotten to delete the sentence about the applcation.

Changed...